txaio/aio.py

```diff
@@ -57,8 +57,9 @@
 config = _Config()
 config.loop = asyncio.get_event_loop()
 _stderr, _stdout = sys.stderr, sys.stdout
-_loggers = []  # weak-ref's of each logger we've created before start_logging()
+_loggers = weakref.WeakSet()  # weak-ref's of each logger we've created before start_logging()
```

Comment:
Uups, ok: sorry to trigger work on your side=( But that particular line seems to be "better" than before.

FWIW, everything above/below (Twisted, Autobahn) doesn't support Python 2.6 anymore, and I wasn't even aware that txaio still works on Py 2.6 - though "accidently", because we don't claim nor test it.

But I've filed https://github.com/crossbario/txaio/issues/65 because `setup.py` classifiers are missing.

cc @hawkowl